### DISCUSSION TITLE
Profile file cannot be null error using DBeaver under Windows10 AWS Athena IAM profile

### DISCUSSION THREAD

**Discussion by sashapoda** [dbeaver/dbeaver#18871] [Q&A]
I've checked this [question on stackoverflow](https://stackoverflow.com/questions/69991535/failure-querying-aws-athena-from-dbeaver-using-iam-profiles-ephemeral-credenti) and [this thread on github](https://github.com/dbeaver/dbeaver/issues/3918#issuecomment-511484596). But it doesn't work for me under Windows 10.
Any ideas?
screenshots:
[https://prnt.sc/JDJ5L4rHaD3p](https://prnt.sc/JDJ5L4rHaD3p)
[https://prnt.sc/_5X_QYpiIesR](https://prnt.sc/_5X_QYpiIesR)
[https://prnt.sc/Kzfxbm0vzruq](https://prnt.sc/Kzfxbm0vzruq)

**Comment by LonwoLonwo** [comment]
Did you try to just use the access key as a user and the secret key as the password in the Athins connection?
